build/config/android/config.gni

```diff
@@ -92,69 +67,13 @@ if (is_android) {
 
   zipalign_path = "$android_sdk_build_tools/zipalign"
 
-  # Subdirectories inside android_ndk_root that contain the sysroot for the
-  # associated platform.
-  if (current_cpu != "x64" && current_cpu != "arm64") {
-    android_api_level = 19
-  }
```

Comment:
I think this needs to stay and get changed to 21. The default above is 22.

Reply:
Good catch.